resources/public/api/conf/2.0/application.yaml

```diff
@@ -0,0 +1,10973 @@
+        </pre>
+                    </td>
+                    <td><p>HTTP status: <code class="code--slim">403 (Forbidden)</code></p>
+        <pre class="code--block">
+        {
+          "code": "INVESTOR_ACCOUNT_ALREADY_OPEN",
+          "message": "The account already has a status of Open"
+        }
+        </pre>
+                    </td>
+                </tr>
+                <tr>
+                    <td><p>Request for an account that is closed with a closure reason as transferred out</p><p class="code--block">lisaManagerReferenceNumber: <a href="#section/Testing">Use your test user profile</a></p></td>
+                    <td>
+        <pre class="code--block">
+        {
+          "accountId": "0000000403"
+        }
+        </pre>
+                    </td>
+                    <td><p>HTTP status: <code class="code--slim">403 (Forbidden)</code></p>
+        <pre class="code--block">
+        {
+          "code": "INVESTOR_ACCOUNT_ALREADY_CLOSED",
+          "message": "You cannot reinstate this account because it was closed with a closure reason of transferred out"
+        }
+        </pre>
+                    </td>
+                </tr>
+                 <tr>
+                    <td><p>Request for an account that is closed with a closure reason as cancelled</p><p class="code--block">lisaManagerReferenceNumber: <a href="#section/Testing">Use your test user profile</a></p></td>
+                    <td>
+        <pre class="code--block">
+        {
+          "accountId": "1000000403"
+        }
+        </pre>
+                    </td>
+                    <td><p>HTTP status: <code class="code--slim">403 (Forbidden)</code></p>
+        <pre class="code--block">
+        {
+          "code": "INVESTOR_ACCOUNT_ALREADY_CANCELLED",
+          "message": "You cannot reinstate this account because it was closed with a closure reason of cancellation"
+        }
+        </pre>
+                    </td>
+                </tr>
+                 <tr>
+                    <td><p>Request for an account that is closed with a closure reason as cancelled</p><p class="code--block">lisaManagerReferenceNumber: <a href="#section/Testing">Use your test user profile</a></p></td>
+                    <td>
+        <pre class="code--block">
+        {
+          "accountId": "3000000403"
+        }
+        </pre>
+                    </td>
+                    <td><p>HTTP status: <code class="code--slim">403 (Forbidden)</code></p>
+        <pre class="code--block">
+        {
+          "code": "INVESTOR_COMPLIANCE_CHECK_FAILED",
+          "message": "You cannot reinstate this account because the investor has failed a compliance check"
+        }
+        </pre>
+                    </td>
+                </tr>
+                <tr>
+                    <td><p>Request containing an account ID that does not exist</p><p class="code--block">lisaManagerReferenceNumber: <a href="#section/Testing">Use your test user profile</a></p></td>
```

Comment:
Once the duplicated Testing section in this file is removed, please update link from `#section/Testing` to `/api-documentation/docs/api/service/lisa-api/2.0#testing`. Thanks